.travis.yml

```diff
@@ -1,12 +1,113 @@
 language: python
+
+sudo: false
+
+env:
+  global:
+    - USE_CONDA=false
+    - COVERAGE=""
+
 python:
+  - 2.7
+  - 3.4
+  - 3.5
   - 3.6
```

Comment:
It looks like a lot has been added to .travis.yml here. I think most of it shouldn't be here.

Try to remove all of these changes, and only change:

- add asv to requrements.txt
- replace calls to runner.py with calls to asv

Reply:
This file still looks like it's the copy from the asv repo. Try reverting to the original version in this repo and only changing one thing: run the `asv run` command instead of `python runner.py`